product-design.md

```diff
@@ -0,0 +1,19 @@
+# Product Design
+
+## Beginning Product Design
+
+* Read [The Lean Startup](http://amzn.com/0307887898).
+* Read [thoughtbot's playbook](http://playbook.thoughtbot.com)
+* Read 37 Signal's [Getting Real](http://gettingreal.37signals.com)
+* Read 37 Signal's [Rework](http://37signals.com/rework)
+* Watch [Ryan Singer's UX Sketching screencast](https://peepcode.com/products/ryan-singer-ux)
+* Watch [Ryan Singer's Prototyping screencast](https://peepcode.com/products/ryan-singer-ii)
```

Comment:
http://www.pluralsight.com/courses/play-by-play-ryan-singer

They both go to the same landing page now.